src/io_uring/src/lib.rs

```diff
@@ -13,6 +15,7 @@
 //! For more information on io_uring, refer to the man pages.
 //! [This pdf](https://kernel.dk/io_uring.pdf) is also very useful, though outdated at times.
 
+#[allow(clippy::undocumented_unsafe_blocks)]
```

Comment:
In the binding modules there is an:
```
#![allow(
    non_camel_case_types,
    non_upper_case_globals,
    dead_code,
    non_snake_case,
    clippy::ptr_as_ptr
)]
```
it would be better to append this to that.

It would also be necessary to update `tools\bindgen.sh` with this allow lint.

Reply:
Applied :)

I ran `bindgen.sh` as well, but the output looks drastically different in ways that are unrelated to my changes (reordered blocks and such).
Should I run and commit the changes? (made the correct changes manually in all generated files)

Reply:
I prefer just the manual fixes without re-running `binden.sh` in this case.